File: cfgs/java_cfgs_foo/PYTHAGOREAN_QUADRUPLE.java
Nodes
1;special;;;
2;statement;int sum = ((foo_1 * foo_2) + (foo_3 * foo_4)) + (foo_5 * foo_6);14;
3;conditional;(foo_1 * foo_2) == foo_3;15;
4;statement;return true;15;
5;exit;;;
6;statement;return false;16;
Edges
0;2;
1;3;
2;4;6;
3;5;
4;
5;5;

File: cfgs/pythoncfgs/PYTHAGOREAN_QUADRUPLE.py
Nodes
1;statement;foo_1 = foo_2 * foo_3 + foo_4 * foo_5 + foo_6 * foo_7;8;
2;conditional;foo_1 * foo_2 == foo_3;9;
3;statement;return false;12;
4;statement;return true;10;
5;exit;;
Edges
0;2;
1;3;4;
2;5;
3;5;
4;